developer_manual/client_apis/OCS/ocs-api-overview.rst

```diff
@@ -1,239 +0,0 @@
-
-Clients can obtain capabilities provided by the Nextcloud server and its apps via the capabilities OCS API.
-
-.. code::
-
-	GET /ocs/v1.php/cloud/capabilities
-
-
-
-.. code:: xml
-
-	<?xml version="1.0"?>
-	<ocs>
-		<meta>
-			<status>ok</status>
-			<statuscode>100</statuscode>
-			<message>OK</message>
-			<totalitems></totalitems>
-			<itemsperpage></itemsperpage>
-		</meta>
-		<data>
-			<version>
-				<major>17</major>
-				<minor>0</minor>
-				<micro>2</micro>
-				<string>17.0.2</string>
-				<edition></edition>
-				<extendedSupport></extendedSupport>
-			</version>
-			<capabilities>
-				<core>
-					<pollinterval>60</pollinterval>
-					<webdav-root>remote.php/webdav</webdav-root>
-				</core>
-			</capabilities>
-		</data>
-	</ocs>
-
-
-Theming capabilities
---------------------
-
-Values of the theming app are exposed through the capabilities API, allowing client developers to adjust the look of clients to the theming of different Nextcloud instances.
-
-.. code:: xml
-
-	<theming>
-		<name>Nextcloud</name>
-		<url>https://nextcloud.com</url>
-		<slogan>A safe home for all your data</slogan>
-		<color>#0082c9</color>
-		<color-text>#ffffff</color-text>
-		<color-element>#0082c9</color-element>
-		<color-element-bright>#aaaaaa</color-element-bright>
-		<color-element-dark>#555555</color-element-dark>
-		<logo>http://cloud.example.com/index.php/apps/theming/logo?v=1</logo>
-		<background>http://cloud.example.com/index.php/apps/theming/logo?v=1</background>
-		<background-plain></background-plain>
-		<background-default></background-default>
-	</theming>
-
-For elements like radio buttons, input borders and more, instead of the primary ``color`` value, the ``color-element-bright`` should be used on bright background and ``color-element-dark`` on dark background.
-This when the primary color is e.g. set to ``#000000`` the ``color-elemenet-dark`` will be set to ``#555555`` so items are still visible. In the Nextcloud web UI only the top header uses ``color``, everything else uses ``color-element-*``.
-Text and icons on these elements should use ``color-text``.
-
-
-The background value can either be an URL to the background image or a hex color value.
-
-Direct Download
----------------
-
-It might be required to give a 3rd party access to a file however you do not
-want to hand over credentials to the 3rd party. An example of this is playing
-files in an external media player on mobile devices.
-
-To solve this issue there is a way to request a unique public link to a single file.
-This link will be valid for 8 hours afterwards it will be removed.
-
-To obtain a direct link:
-
-.. code::
-
-	POST /ocs/v2.php/apps/dav/api/v1/direct
-
-With the :code:`fileId` in the body (so :code:`fileId=42` for example).
-This will then return you the link to use to obtain the file.
-
-Notifications
--------------
-
-There is also the `Notifications API <https://github.com/nextcloud/notifications/blob/master/docs/ocs-endpoint-v2.md>`_
```

Comment:
Should also be kept

Reply:
The notifications API is well documented using OpenAPI. I can add all the extra information from the markdown docs to the code so it'll show up.

Reply:
Notifications is not part of the spec yet, so I'll revert it (it is documented but not generated).